cypress/tests/cli_test/BUILD

```diff
@@ -5,7 +5,16 @@ cypress_test(
     args = [
         "run",
         "--config-file=cypress.config.js",
-    ],
+    ] + select({
+        "@bazel_tools//src/conditions:darwin": [],
+        "//conditions:default": ["--browser=../../../../chrome_linux/chrome"],
+    }),
+    browsers = select({
+        # Chrome for mac contains files with spaces which are bazel incompatible.
```

Comment:
hmmm. files with spaces are ok if they are behind a directory artifact. I think we can make it work in a follow-up.

Reply:
Sounds good, I'd like that. Probably doing a proper toolchain is what we want in this case